tests/home-page-filters.spec.ts

```diff
@@ -6,19 +6,19 @@ test('check if filters are saved inside the localStorage', async ({ page }) => {
   await homePage.go();
   await homePage.importData();
 
-  await homePage.checkNumberOfDisplayedItems(16, 16);
+  await homePage.checkNumberOfDisplayedItems(17, 17);
```

Comment:
suggestion: could refactor seeder to expose seedData thne inject seedData.size in checkNumberOfDisplayedItems second argument for consistency